Update the installation guide (#105)
glotzerlab/signac-docs: docs/source/installation.rst

.. _conda-forge: https://conda-forge.github.io
.. _pip: https://docs.python.org/3/installing/index.html

The **signac** framework consists of three packages namely, **signac**, **signac-flow**, and **signac-dashboard** (See :ref:`package-overview` for more information).

> All packages in the **signac** framework depend on the core **signac** package, which provides the data management functionality used by all other packages

Reviewer: ```suggestion
All packages in the **signac** framework depend on the core **signac** package, which provides the data management functionality used by all other packages.
```

.. rubric:: `signac (core) <signac-core_>`_

> - Implement your own, file based database.
> - Search and manipulate the data in existing **signac** databases.
> - Provides the data management functionality for **signac-flow** and **signac-dashboard**.

Reviewer: ```suggestion
- Manage project data with a well-defined indexable storage layout for data and metadata.
- Search and manipulate the data in existing **signac** projects.
- Provides the data management functionality for **signac-flow** and **signac-dashboard**.
```

.. rubric:: signac-flow_

> - Implement workflows on the databases managed with **signac**.
> - Run complex workflows from the command line.
> - Submit jobs to high-performance clusters (HPC).
> - Requires the core **signac** package.

Reviewer: ```suggestion
- Implement workflows for projects managed with **signac**.
- Run complex workflows from the command line.
- Submit jobs to high-performance clusters (HPC).
- Requires the core **signac** package.
```

.. rubric:: signac-dashboard_

> - Share workspaces through a browser-based GUI.
> - Requires the core **signac** package.

Reviewer: ```suggestion
- Share workspaces through a browser-based GUI.
- Visualize job data such as text, images, or video.
- Requires the core **signac** package.
```
